apps/ashley/vars/settings.yml

```diff
@@ -0,0 +1,3 @@
+databases:
+  - engine: "postgresql"
+    release: "12"
```

Comment:
Sure? Will it work with potsgresql 9.6?

Reply:
Our production is running posgresql 12.

Reply:
Yes, but this is a (unnecessary?) strong requirement and not consistent with other apps like [richie](https://github.com/openfun/arnold/blob/master/apps/richie/vars/settings.yml#L3) or [marsha](https://github.com/openfun/arnold/blob/master/apps/marsha/vars/settings.yml#L3)

Reply:
Ah ok. I misunderstood your comment.

Reply:
Thanks for your feedbacks :slightly_smiling_face: 

:man_shrugging: I do not realize all the consequences that this implies to set version 12 here. It defines a requirement, or just a variable that one can override if he wants to use another version ?

I use postgresql 12 in `ashley` since [this suggestion](https://github.com/openfun/ashley/pull/3#discussion_r385197418), so I put it everywhere to be consistent. Do I have to be consistent with other projects, or ashley ? :slightly_smiling_face:

Reply:
In case someone else wants to deploy it at home, we need to ensure the largest compatibility with database engines.

Reply:
I've set version `9.6`, but I'm not sure this is the solution to ensure the largest compatibility with database engines. Now we don't know if it works on latest releases.

To achieve this, another approach would be to run the CI on multiple database versions in ashley project, but it's out of the scope of this PR.